docs/pages/distribution/release-channels.md

```diff
@@ -103,7 +103,7 @@ Environment variables don't exist explicitly, but you can utilize release channe
 
 Say you have a workflow of releasing builds like this:
 
-<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2']} />  
+<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2']} />  
```

Comment:
```suggestion
<TerminalBlock cmd={[
  '# Publish to release channel prod-v1',
  'expo publish --release-channel prod-v1',
  '',
  '# Publish to release channel prod-v2',
  'expo publish --release-channel prod-v2',
  '',
  '# Publish to release channel prod-v3',
  'expo publish --release-channel prod-v3', 
  '', 
  '',
  '# Publish to release channel staging-v1', 
  'expo publish --release-channel staging-v1',
  '', 
  '# Publish to release channel staging-v2',
  'expo publish --release-channel staging-v2'
]} />  
```

Let's improve the readability of this example for the editors, since we touch those lines anyway. 🙂